pkg/service/backup/restore.go

```diff
@@ -244,3 +254,29 @@ func (s *Service) forEachRestoredManifest(clusterID uuid.UUID, target RestoreTar
 		return s.forEachManifest(ctx, clusterID, []Location{location}, filter, f)
 	}
 }
+
+// listAllViews is the utility function that queries system_schema.views table to get list of all views created on the cluster.
+func (s *Service) listAllViews(ctx context.Context, clusterID uuid.UUID) (views []string, err error) {
```

Comment:
Maybe it's worth to mention that the `system_schema.views` also contains indexes?
It wasn't clear for me why we only query this table and not the `system_schema.indexes`.

Reply:
Index actually is a view that shows the table ordered according to the index definition.
But I can add comment about that anyway.